packages/dcos-log/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.7
```

Comment:
updated PR to use base image and added a commit from https://github.com/dcos/dcos/pull/998